ctapipe/image/muon/muon_integrator.py

```diff
@@ -407,7 +418,7 @@ def fit_muon(self, centre_x, centre_y, radius, pixel_x, pixel_y, image):
         init_constrain['fix_centre_x'] = True
         init_constrain['fix_centre_y'] = True
         init_constrain['limit_ring_width'] = (0., 1.)
-        init_constrain['limit_optical_efficiency_muon'] = (0., 1.)
+        #init_constrain['limit_optical_efficiency_muon'] = (0., 1.) # Unneeded constraint - and misleading in case of changes leading to >1 values!
```

Comment:
it should still be > 0

Reply:
> it should still be > 0

It is an unneeded constraint, given the nature of the fit (the factor is the same for the whole event). If we ever get a negative number there, it would tell us immediately something is very wrong.

Reply:
I made the experience that minimizers need these constraints, as they won't be able to recover from invalid values once they have tried them out.